pages/pycon-landing.tsx

```diff
@@ -6,42 +6,70 @@ import SponsorComponent from "../components/sponsor";
 import CommunityPartners from "../components/communityPartners";
 import SubscribeSection from "../components/subscribe";
 import RegistrationSection from "../components/registration";
-import ConferenceSchedule from "../components/schedule";
 import Partners from "components/partners";
 import LocalUserGroupsChapters from "components/localUserGroups";
 import { fetchSpeakers } from "lib/data";
 import Speakers from "components/speakers";
 import VenueSection from "../components/venue";
+import Link from "next/link";
+import IconComponent from "@components/icons";
 export default function PyConIndiaWeb() {
   const speakers = fetchSpeakers();
   return (
     <div>
       <Header />
       <HeroBanner />
       <JourneyComponent />
-      <Speakers speakers={speakers["keynoteSpeakers"]} showMicCreative={true} title="Keynote Speakers" />
-      <ConferenceSchedule />
+      <Speakers
+        speakers={speakers["keynoteSpeakers"]}
+        showMicCreative={true}
+        title="Keynote Speakers"
+      />
+      {/* Schedule CTA starts */}
+      <div className="bg-conference home-section pt-4">
+        <div className="container">
+          <div className="row pt-5 pb-5">
+            <div className="col-md-12 text-center">
+              <Link
+                href="/schedule/"
+                className="text-decoration-none text-light"
+              >
+                <button className="custom-button green-btn">
```

Comment:
The text should be **Check out the Schedule** as we are not showing partial schedule here.
Can we use Button component here? It also accepts icon name.